README.md

```diff
@@ -16,18 +20,23 @@ Consider setting the git option `submodule.recurse=true`.
 
 - devnet: 4MangoMjqJ2firMokCjjGgoK8d4MXcrgL7XJaL3w6fVg
 - mainnet-beta: 4MangoMjqJ2firMokCjjGgoK8d4MXcrgL7XJaL3w6fVg
+- primary mango group on mainnet-beta: 78b8f4cGCwmZ9ysPFMWLaLTkkaYnUjwMJYStWe5RTSSX
+
+### Release
+
+Here are steps followed while performing a program deployment to mainnet-beta
 
-### Notes
+- review diff of last deployed tag to mainnet-beta, e.g. https://github.com/blockworks-foundation/mango-v4/compare/program-v0.4.0..dev, pay special attention to account layout changes, backward compatibility of newly introduced account fields, etc.
+- update changelog
+- add a git tag e.g. `program-v0.0.1`, should match the version the program has
+- deploy to mainnet-beta
```

Comment:
typically I deploy first, then put the deploy timestamp/tx info into the changelog and only then tag the version